lib/CompilerInterfaceTypes.h

```diff
@@ -14,6 +14,19 @@
 #include <cstring>
 #include <stdint.h>
 
+namespace proteus {
+
+enum RuntimeConstantTypes : int32_t {
+  BOOL = 1,
```

Comment:
This is a great addition, I should've done this in the first place

Reply:
Yes, to match the expected data type from the pass